pkg/pods/hoist_launchable.go

```diff
@@ -130,19 +130,13 @@ func (hoistLaunchable *HoistLaunchable) Stop(serviceBuilder *runit.ServiceBuilde
 // All services will attempt to be started.
 func (hoistLaunchable *HoistLaunchable) Start(serviceBuilder *runit.ServiceBuilder, sv *runit.SV) error {
 
-	// if the service is new, building the runit services also starts them, making the sv start superfluous but harmless
-	err := hoistLaunchable.BuildRunitServices(serviceBuilder)
-	if err != nil {
-		return err
-	}
-
 	executables, err := hoistLaunchable.Executables(serviceBuilder)
 	if err != nil {
 		return err
 	}
 
 	for _, executable := range executables {
-		_, err := sv.Restart(&executable.Service)
+		_, err := sv.Start(&executable.Service)
```

Comment:
We don't stop the service before this point. We rely on restart to atomically stop and start - this is required to do the self-hosted preparer. Otherwise, you'd run `sv stop` on the preparer, which would kill the launch process.